packages/design-system-tailwind-preset/CHANGELOG.md

```diff
@@ -7,6 +7,23 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [0.5.0]
+
+### Added
+
+- Added support for monochromatic button design tokens ([#709](https://github.com/MetaMask/metamask-design-system/pull/709))
+- Added background color tokens: section, subsection, muted ([#682](https://github.com/MetaMask/metamask-design-system/pull/682))
+- Added DisplayLG typography token support ([#607](https://github.com/MetaMask/metamask-design-system/pull/607))
+
+### Changed
+
+- **BREAKING:** Updated peer dependency to `@metamask/design-tokens@^8.0.0` which includes several breaking changes:
+  - Font family tokens now use base font families (`--font-family-default`, `--font-family-accent`, `--font-family-hero`) instead of individual typography variant font families ([#533](https://github.com/MetaMask/metamask-design-system/pull/533))
+  - Primary font changed from Euclid Circular B to CentraNo1 ([#499](https://github.com/MetaMask/metamask-design-system/pull/499))
```

Comment:
This should be changed to Geist right?

Reply:
Yes. Good catch!

Reply:
Updated https://github.com/MetaMask/metamask-design-system/pull/754/commits/618e591500c04764e0241135fe4687539d6cd33a